images/win/scripts/Installers/Validate-Haskell.ps1

```diff
@@ -0,0 +1,52 @@
+################################################################################
+##  File:  Validate-Haskell.ps1
+##  Desc:  Validate Haskell for Windows
+################################################################################
+
+# GHC validation
```

Comment:
Could you please validate for each ghc version that we can run a simple command `ghc --version`?